keyboards/electronlab/klor/config.h

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2024 ElectronLab
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* The way how "handedness" is decided (which half is which), see https://docs.qmk.fm/#/feature_split_keyboard?id=setting-handedness for more options.*/
+//#define EE_HANDS
+#define MASTER_LEFT
+//#define MASTER_RIGHT
+//#define SPLIT_HAND_PIN GP10 //If handedness jumpers are bridged on KLOR REV 1.5 PCB onward
+
```

Comment:
Should I remove all handedness options even the failproof one ? This will make the split keyboard unresponsive upon initial FW flashing unless forcefully corrected

Reply:
The only commented out is `MASTER_LEFT`, which is the default behaviour. So if removing these fives lines breaks something, something else is wrong.

Reply:
Ah all right I didn't get that this was the default behaviour that's why !!

Reply:
Agreed. Change is done and committed.